src/common/hist_util.cu

```diff
@@ -94,6 +94,17 @@ __global__ void UnpackFeaturesK
 struct SketchContainer {
   std::vector<HistCutMatrix::WXQSketch> sketches_;  // NOLINT
   std::vector<std::unique_ptr<std::mutex>> col_locks_; // NOLINT
+  SketchContainer(const tree::TrainParam &param, DMatrix *dmat) {
+    const MetaInfo &info = dmat->Info();
+    // Initialize Sketches for this dmatrix
+    sketches_.resize(info.num_col_);
+    col_locks_.resize(info.num_col_);
+#pragma omp parallel for schedule(static) if (info.num_col_ > 1000)
```

Comment:
This number occurs in more than one place. Consider defining an constant and using it instead.

Reply:
done